styles.css

```diff
@@ -686,23 +686,20 @@ header {
 
 .header__image img {
   position: absolute;
-
   top: 50%;
   left: 75%;
-  border: 0.5rem solid var(--img-color);
+  border: 0.2rem solid black; /* Set border color to black and reduce width */

```

Comment:
![image](https://github.com/user-attachments/assets/7e38c3cd-3338-445d-bd4d-c9fccef737a4)

like after the changes u have made with the border color & as well with the bg the color scheme of all the headings has been looking like this , try to solve this ASAP within tom & push your commits